test/e2e/dra/test-driver/app/kubeletplugin.go

```diff
@@ -315,15 +316,17 @@ func (ex *ExamplePlugin) nodePrepareResource(ctx context.Context, claimReq *drap
 
 	var devices []Device
 	for _, result := range claim.Status.Allocation.Devices.Results {
-		requestName := result.Request
+		fullRequestName := result.Request
+		segments := strings.Split(fullRequestName, "/")
+		baseRequestName := segments[0]
```

Comment:
So, this is saying: The CDI info only includes main request name, but the driver still can tell which subrequest though, and make decisions based on that if it cares. Correct?

Reply:
So with this change, the `request` field in `DeviceRequestAllocationResult` will contain either just a reference to a request or a reference to a subrequest (on the format `<parent request>/<subrequest>`). The test driver uses the value of this field for a few things, including determining the configs that should be used, for setting the environment in the container, and for constructing the path for the CDI file. The first of these requires that the driver can tell which subrequest was chosen, since configs might be specific to a single subrequest.

I'm not sure whether the environment variables set in the container or the path for the CDI file is part of a spec or just a choice in the test driver. I also had to make changes to the handling in the driver, so I'm wondering if this might become a breaking change in existing drivers (of which there are obviously very few...). I wanted some input from @pohly on this.

Reply:
This was answered in https://github.com/kubernetes/kubernetes/pull/128586#discussion_r1886253818, so resolving this comment.